src/tests/test_sexual_behaviour.py

```diff
@@ -71,18 +93,23 @@ def check_num_partners(row):
 
 def test_num_partners():
     """Check that number of partners are reasonable"""
-    pop_data = Population(size=1000, start_date=date(1989, 1, 1)).data
-    assert(any(pop_data["num_partners"] > 0))
+    pop = Population(size=100000, start_date=date(1989, 1, 1))
+    # t1 = time.perf_counter()
+    # for x in range(0, 500):
```

Comment:
Remove these (and the call to `import time`) instead of commenting, or is performance still a concern?